### PR TITLE
Generate from Navigation Drawer template.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
Created app skeleton from Navigation Drawer new project in Android Studio. Tuned the .gitignore slightly.

@passy @pautomas

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,10 @@ proguard/
 
 # Android Studio captures folder
 captures/
+
+# Android Studio project files
+*.iml
+.idea/
+
+# Non project files
+.DS_Store
```
Comment (passy) [inline .gitignore]: I prefer these platform-specific one in a global ignore config, but I'll let this one slide.

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "19.1.0"
```
Comment (passy) [inline app/build.gradle]: Surprised by this. `23.0.1` is the most recent. Did AS generate that?

Comment (passy) [inline app/build.gradle]: PUPDATE: `23.0.3` is the most recent stable one, `24rc2` is in preview.

File: gradlew.bat [hunk 3]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (passy) [inline gradlew.bat]: Haha, I always delete that Windows rubbish. I mean, they even have bash now.

File: gradlew [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        (0) set -- ;;
+        (1) set -- "$args0" ;;
+        (2) set -- "$args0" "$args1" ;;
+        (3) set -- "$args0" "$args1" "$args2" ;;
+        (4) set -- "$args0" "$args1" "$args2" "$args3" ;;
+        (5) set -- "$args0" "$args1" "$args2" "$args3" "$args4" ;;
+        (6) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" ;;
+        (7) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" ;;
+        (8) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" ;;
+        (9) set -- "$args0" "$args1" "$args2" "$args3" "$args4" "$args5" "$args6" "$args7" "$args8" ;;
```
Comment (passy) [inline gradlew]: Makes me smile every time.

File: gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#Wed Oct 21 11:34:03 PDT 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-all.zip
```
Comment (passy) [inline gradle/wrapper/gradle-wrapper.properties]: 2.12 is the latest stable, but I'm not sure if it's compatible. They've done some good performance work, might be worth trying.

**Comment by passy** [comment]
#shipit :shipit:

File: gradlew (hunk 4 above)
Comment (daithiocrualaoich) [inline gradlew]: WAT!

File: gradlew.bat (hunk 3 above)
Comment (daithiocrualaoich) [inline gradlew.bat]: Done.

File: app/build.gradle (hunk 2 above)
Comment (daithiocrualaoich) [inline app/build.gradle]: Done. Good pupdate.

**Pull request merged**